support-lib/cx/CppWrapperCache.h

```diff
@@ -0,0 +1,86 @@
+//
```

Comment:
Can the Cpp and Cx proxy caches be refactored to use the shared template impl in proxy_cache_*?